README.md

```diff
@@ -9,47 +9,49 @@ TRAM enables researchers to test and refine Machine Learning (ML) models for ide
 Through research into automating the mapping of cyber threat intel reports to ATT&CK, TRAM aims to reduce the cost and increase the effectiveness of integrating ATT&CK into cyber threat intelligence across the community. Threat intel providers, threat intel platforms, and analysts should be able to use TRAM to integrate ATT&CK more easily and consistently into their products.
 
 ## Table of contents
+
 - [TRAM](#tram)
   - [Table of contents](#table-of-contents)
   - [Installation](#installation)
   - [Installation Troubleshooting](#installation-troubleshooting)
     - [[97438] Failed to execute script docker-compose](#97438-failed-to-execute-script-docker-compose)
   - [Requirements](#requirements)
-  - [Developer Setup](#developer-setup)
+  - [For Developers](#for-developers)
     - [Makefile Version](#makefile-version)
       - [Useful User Commands](#useful-user-commands)
       - [Useful Dev Commands](#useful-dev-commands)
+    - [Developer Setup](#developer-setup)
```

Comment:
Is this just getting moved? Or was this meant to be For Developers?

Reply:
just re-orged the readme a bit